user-management-application/src/main/java/com/user/management/app/facade/employee/Impl/EmployeeFacadeImpl.java

```diff
@@ -0,0 +1,29 @@
+package com.user.management.app.facade.employee.Impl;
+
+
+import com.user.management.app.facade.employee.EmployeeFacade;
+import com.user.management.app.model.employee.Employee;
+import com.user.management.data.employee.dao.impl.employeeDaoImpl;
+
+import java.sql.SQLException;
+
+public class EmployeeFacadeImpl implements EmployeeFacade {
+    private final com.user.management.data.employee.dao.employeeDao employeeDao;
+
+    public EmployeeFacadeImpl(com.user.management.data.employee.dao.employeeDao employeeDao) {
+        this.employeeDao = employeeDao;
+    }
+
+    public EmployeeFacadeImpl() {
+        this.employeeDao = new employeeDaoImpl();
+    }
+
+    @Override
+    public Employee saveEmployee(Employee employee) throws SQLException {
+        return employeeDao.saveEmployee(employee);
+    }
+    @Override
+    public Employee checkEmployeeId(String employeeId) throws SQLException {
+        return employeeDao.checkEmployeeId(employeeId);
```

Comment:
why is the name of the method like this? checkEmployeeId method implies to check if the employee id exists.
refactor accordingly. do the same with other methods.

Reply:
I've refactored it accordingly sir